stubs/SQLAlchemy/sqlalchemy/dialects/mysql/asyncmy.pyi

```diff
@@ -54,7 +54,7 @@ class AsyncAdapt_asyncmy_dbapi:
     asyncmy: Any
     pymysql: Any
     paramstyle: str
-    def __init__(self, asyncmy, pymysql) -> None: ...
+    def __init__(self, asyncmy: Any) -> None: ...
```

Comment:
Why is this annotated with `Any` as opposed to not being annotated at all?

Reply:
This gets passed the `asyncmy` module. I don't think we have a proper way to annotate this, except for maybe using `ModuleType`, which doesn't seem right either.